### PR TITLE
Mitigate QS Vulnerability

### PR DESCRIPTION
**Pull request by TommyJ1994**
Mitigate QS vulnerability (6.4.0)  https://issues.jboss.org/browse/RAINCATCH-603

Verified manually that the vulnerability is mitigated using `snyk test`

**3rd Party Module Fix Status**
-  request  -  2.81.0

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "unifiedpush-node-sender",
-  "version": "0.15.0",
+  "version": "0.16.0",
```
Comment (lholmquist) [inline package.json]: i don't think this should be updated in the PR.  it will be updated during the release

Comment (TommyJ1994) [inline package.json]: Ok, i can update the PR to remove this version bump. When is the next release do you know? I will need to reference the vulnerability-free version of this module in another RHMAP component.

Comment (lholmquist) [inline package.json]: once this PR is merged i can do a release.

Comment (lholmquist) [inline package.json]: probably will need to regenerate that npm shrinkwrap file too

Comment (TommyJ1994) [inline package.json]: Removed Version Bump, Regenerated Shrinkwrap.

**Comment by lholmquist** [comment]
@TommyJ1994 landed 🚀

**Comment by TommyJ1994** [comment]
@lholmquist :champagne:

**Comment by lholmquist** [comment]
@TommyJ1994 also published,  so you should be good now :)

**Comment by TommyJ1994** [comment]
@lholmquist Fantastic! Thanks for speedy review and publish!

**Pull request merged**